specification/ws.extensions/x-kaazing-ping-pong/src/main/scripts/org/kaazing/specification/ws.extensions/x-kaazing-ping-pong/wsn/server.may.send.extended.ping.control.frames/request.rpt

```diff
@@ -0,0 +1,41 @@
+connect tcp://127.0.0.1:8001
```

Comment:
This test is testing inactivity timeout, so please rename to something like "server.should.timeout.if.client.does.not.respond.to.extended.ping"